jupyter_server/auth/identity.py

```diff
@@ -290,11 +292,28 @@ def user_to_cookie(self, user: User) -> str:
         Default is just the user's username.
         """
         # default: username is enough
```

Comment:
This comment and the docstring are now out of date :) I think also we should clarify whether this cookie format is now a required, de-facto standard by JupyterLab (will lab stop working if this cookie structure is broken?).